app/src/main/java/org/hwyl/sexytopo/model/survey/Station.java

```diff
@@ -47,7 +47,7 @@ public String getName() {
     }
 
     public void setName(String name) {
-        this.name = name;
+        this.name = sanitiseName(name);
```

Comment:
I'm not sure on the most-idiomatic conventions for Java, but the constructors could be modified to call this setting; instead of repeating their calls to `sanitiseName`.

Reply:
I think the convention is usually to set stuff directly in the constructor, but makes sense to call setFoo if there's extra logic in there.